src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -144,6 +147,8 @@ def from_compression_config(
 
         sparsity_config = cls.parse_sparsity_config(compression_config)
         quantization_config = cls.parse_quantization_config(compression_config)
+        # NOTE: transfrom config is not support by ctconfig yet
```

Comment:
nit: better as a TODO? or is this removed after CT branches are merged?

Reply:
maybe change it to CompressedTensorsConfig?